api/v1/stats/publications/PKPStatsPublicationHandler.php

```diff
@@ -130,16 +140,19 @@ public function getMany(SlimHttpRequest $slimRequest, APIResponse $response, arr
         $defaultParams = [
             'orderDirection' => StatisticsHelper::STATISTICS_ORDER_DESC,
         ];
-        $initAllowedParams = [
-            'dateStart',
-            'dateEnd',
-            'count',
-            'offset',
-            'orderDirection',
-            'searchPhrase',
-            $this->sectionIdsQueryParam,
-            'submissionIds',
-        ];
+        $initAllowedParams = array_merge(
+            [
+                'dateStart',
+                'dateEnd',
+                'count',
+                'offset',
+                'orderDirection',
+                'searchPhrase',
+                $this->sectionIdsQueryParam,
+                'submissionIds',
+            ],
+            $this->getAppSpecificParams()
```

Comment:
Rather than adding an extra function to just get the app params, maybe make a base function and extend it in an app as needed? Like:

```php
// PKP\API\v1\stats\publications
protected function getManyAllowedParams() {
    return [
        //
    ];
}
protected function getTimelineAllowedParams() {
    return [
        //
    ];
}
```

```php
// APP\API\v1\stats\publications
protected function getManyAllowedParams() {
    $params = parent::getManyAllowedParams();
    $params[] = 'issueIds';
    return $params;
}
protected function getTimelineAllowedParams() {
    $params = parent::getTimelineAllowedParams();
    $params[] = 'issueIds';
    return $params;
}
```

```php
// example use in the handler
$initAllowedParams = $this->getManyAllowedParams();
```